wordpress-mobile/gutenberg-mobile
# "Add Block" icon color

fixes: #546

This PR implements the changes made in https://github.com/WordPress/gutenberg/pull/14631 to change the default color of Dashicons (in this case on `ToolbarButton`).

![img_1268](https://user-images.githubusercontent.com/9772967/53107540-80166180-3535-11e9-93a2-05cb51e7d2e7.png)

To test:
- Build the app.
- Check that the add-block button is filled with `$blue-wordpress` color.
